provider/asdf/asdf.go

```diff
@@ -64,6 +64,7 @@ func (a *AsdfToolProvider) InstallTool(tool provider.ToolRequest) (provider.Tool
 			}
 			resolution, err = ResolveVersion(tool, releasedVersions, installedVersions)
 			if err != nil {
+				// TODO: better error here
```

Comment:
Follow-up PR :) If there are not strictly matching versions, but we see that the requested version is similar to one of the released versions, we could suggest `version:latest` and `version:installed`